### PR TITLE
Chk 772 load test pagopa payment methods

### PR DESCRIPTION
**Pull request by Fernando-Granato**
Below a list of the changes:

Add soak tests for ecommerce-payment-methods service module:
        - Tests for psp API (GET all)
        - Tests for payment methods API (GET POST PATCH)

### CODE REVIEW

File: src/soak-test/soak-test-payment-methods.ts [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+            rate: config.rate, // e.g. 20000 for 20K iterations
+            duration: config.duration, // e.g. '1m'
+            preAllocatedVUs: config.preAllocatedVUs, // e.g. 500
+            maxVUs: config.maxVUs, // e.g. 1000
+        },
+    },
+    thresholds: {
+        http_req_duration: ["p(99)<1500"], // 99% of requests must complete below 1.5s
+        checks: ['rate>0.9'], // 90% of the request must be completed
+        "http_req_duration{api:checkMessages}": ["p(95)<1000"],
```
Comment (pietro-tota) [inline src/soak-test/soak-test-payment-methods.ts:21]: change here this check. the specified "checkMessages" label does not be used into test calls.
Put one entry of this check for each tags used into the below calls such as 'create-payment-method-test', 'patch-status-payment-method-test', etc

File: src/soak-test/soak-test-payment-methods.ts [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        response,
+        { "Response status from POST /payment-methods was 200": (r) => r.status == 200 },
+        { api: "create-payment-method-test" }
+    );
+
+    if (response.status == 200) {
+        paymentMethodId = response.json()["id"];
+        //Test for GET single payment method after POST
+        url = `${urlBasePath}/checkout/ecommerce/v1/payment-methods/${paymentMethodId}`;
+        response = http.get(url);
```
Comment (pietro-tota) [inline src/soak-test/soak-test-payment-methods.ts:76]: add missing get-single-payment-method-test tag into get request

File: src/soak-test/soak-test-payment-methods.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+        );
+
+        if (response.status == 200) {
+            paymentMethodStatus = response.json()["status"];
+
+            if (paymentMethodStatus === "ENABLED") {
+
+                //Test for GET psps by paymentMethodId
+                url = `${urlBasePath}/checkout/ecommerce/v1/payment-methods/${paymentMethodId}/psps`;
+                response = http.get(url);
```
Comment (pietro-tota) [inline src/soak-test/soak-test-payment-methods.ts:104]: add missing tag into get, use the same used into check

File: src/soak-test/soak-test-payment-methods.ts [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+        }
+    } else {
+        fail('Error creating new payment method');
+    }
+
+
+
+    //Test for GET all payment method
+    url = `${urlBasePath}/checkout/ecommerce/v1/payment-methods`;
+    response = http.get(url);
```
Comment (pietro-tota) [inline src/soak-test/soak-test-payment-methods.ts:125]: same as above

File: src/soak-test/soak-test-payment-methods.ts [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+    check(
+        response,
+        { "Response status from GET payment-methods was 200": (r) => r.status == 200 },
+        { api: "get-all-payment-methods-test" }
+    );
+
+    //Test for GET psps
+    url = `${urlBasePath}/checkout/ecommerce/v1/payment-methods/psps`;
+    response = http.get(url);
```
Comment (pietro-tota) [inline src/soak-test/soak-test-payment-methods.ts:134]: same as above

**Review by pietro-tota — COMMENTED** [review]
Well done, fix some tests tags so that request and response checks are correctly linked

**Review by MarcoAGreco — APPROVED** [review]

**Pull request merged**